src/pages/patterns/notification-pattern/index.mdx

```diff
@@ -186,6 +194,7 @@ Banners take over the top of an interface to show notifications general to the p
 #### Best practices:
 
 - Banners should be placed at the top of the content area they relate to.
+- Do not cover other content with a banner notification.
 - Place system-wide messages directly below the main header or navigation bar.
 - Banners are not sticky and should scroll with the other content on the page.
 - Only show one banner at a time.
```

Comment:
>Carbon currently does not have a banner component, but if your team has developed one it can be contributed and made available to others.

Since we do have a banner (styled exactly the same as inline notification), should this line be removed?

Reply:
Good question! We're going to clarify the language about not having the component in Carbon - the inline banner is just an example to show the idea, but we haven't done enough iteration and testing to define it as a banner component.